### ISSUE TITLE
Can not log in - Firefox or Chrome in Ubuntu (16.04)

### ISSUE THREAD

**Issue by osmCL** [maproulette/maproulette3#313]
You click the "Sign in" button in the Maproulette 3 start page, it takes you to the OSM OAuth page, you grant access, it takes you back to the Maproulette 3 site, but nothing happens, you are still logged out.

**Comment by mvexel** [comment]
When you use OAuth sign in through OSM with other applications, does that work OK? For example http://level0.osmz.ru ?

**Comment by mvexel** [comment]
It works fine for me, however I am on Xubuntu 18.04. Firefox 59.0.2
Can you see if there's any errors in the browser console, or any network resources that fail to load?
Any extensions that could be blocking the process?

**Comment by osmCL** [comment]
I am getting this after the OAuth authorization: 
![mp3_01](https://user-images.githubusercontent.com/28224747/38880580-71b04e18-423c-11e8-83f8-57bb63ab1eff.png)
![mp3_02](https://user-images.githubusercontent.com/28224747/38880581-71cf55b0-423c-11e8-8380-333d85f7c5ad.png)

**Comment by mvexel** [comment]
I was able to reproduce this. Logging out of OSM and logging in again solved it for me. Really odd.

**Comment by mvexel** [comment]
@osmCL Can you confirm that logging out of OSM and then trying again makes this work?

**Comment by osmCL** [comment]
No, it is not working for me. I tried going to the OSM site log off and log in, then tried to authorize MR3. It did not work. After that I went to the OSM site, logged off, went to MR3, tried to authorize, logged in to OSM at that point, but that didn't work either. Can you give me a step by step test sequence? Maybe I am doing something slightly different.

**Comment by osmCL** [comment]
I just found some really strange "explanation" for the problem. Today I clicked the mr3 leaderboard link in Maproulettes tweet, the mr3 site showed me the leaders (http://maproulette.org/mr3/leaderboard), I clicked the "Sign in" button and It worked! Great! I went looking for the tweet with the link to the the challenge that we are working right now here in Chile (2986), clicked it and I am logged out again, try to sign in and nothing happens (same behavior that motivated this issue). 
I try to see the difference and noticed that todays leaderboard sequence started with maproullete.org and the link to our challenge is written as www.maproulette.org... so all the problem seems to be related with using or not using the www at the beggining of URLs.

**Comment by nrotstan** [comment]
Wow, great find! I can replicate it with the www.

**Comment by doublah** [comment]
Still a problem here, www. doesn't work, yet without does.

**Comment by nrotstan** [comment]
Thanks for the feedback, @MRVDH! Can you elaborate a bit on the glitchiness you're experiencing at maproulette.org (not www)? Is logging in not always working for you? And just to confirm, you're solely using maproulette.org and not trying to mix it with www, correct?

We did have an unrelated issue (#427) over the weekend that prevented successful login, which was fixed yesterday. I apologize if that impacted you.

But regarding this issue, `www.maproulette.org` isn't supported and we need to redirect it to maproulette.org to avoid confusion.

**Comment by MRVDH** [comment]
I only use the non-www version because I can not log in with the www version. Logging in there always works but I always have to log in via the leaderboard page because every other page redirects me to the www version. Once I log in via the non-wwe version leaderboard page I can navigate to other pages without any problem. 

However, every day I have to log in again.

**Comment by osmCL** [comment]
I am experiencing similar problems, the login only working in the non-www
version of the site.

Best Regards,

Julio Costa Zambelli
Fundación OpenStreetMap Chile

julio.costa@openstreetmap.cl

https://www.openstreetmap.cl/
Cel: +56(9)89981083
